trx_examples/streaming/LTE_PA_App/@LTEAppInternals/LTEAppInternals.m

```diff
@@ -0,0 +1,348 @@
+                   PDSCHevm_temp = ScEVM.evmGrid(:);
+                   app.LTEAppInternalsProp.PDSCHevm = mean(PDSCHevm_temp(PDSCHevm_temp~=0));
+                   app.LTEAppInternalsProp.evmRB = ...
+                       struct('RMS', RbEVM.evmRBRMS, 'Peak', RbEVM.evmRBPeak);  
+                   app.LTEAppInternalsProp.evmSymbol = ...
+                       struct('RMS', SymbEVM.evmSymbolRMS, 'Peak', SymbEVM.evmSymbolPeak);
+                   
+                   if (mod(i, 10)==9 || (nFrames==0 && i==nSubframes-1))                       
+                       app.LTEAppInternalsProp.FrameEVM = ...
+                           struct('Low', frameLowEVM, ...
+                           'High', frameHighEVM, 'Overall', frameEVM);                                       
+                   end    
+                   
+                   if (i == 0)
+                       app.SummaryTable1.Data(:, 2) = app.SummaryTable1_Data;
+                   end                   
+                   
+                   app.SummaryTable2.Data(:, 3) = app.SummaryTable2_Data;
+                   if (i == nSubframes-1)
+                       temp_SummaryTable3 = cell(8, 1);
+                       for ii = 1:8
+                           app.SummaryTable3_Data(ii, 3) = ...
+                               app.SummaryTable3_Data(ii, 1)/app.SummaryTable3_Data(ii, 2);
+                           if strcmp(app.dBPercentDropDown.Value, 'dB')
+                               temp_SummaryTable3{ii} = ...
+                                   sprintf('%2.3f', 20*log10(0.01*app.SummaryTable3_Data(ii, 3)));
+                           else
+                               temp_SummaryTable3{ii} = sprintf('%2.3f', app.SummaryTable3_Data(ii, 3));
+                           end
+                       end
+                       app.SummaryTable3.Data(:, 2) = temp_SummaryTable3;
+                       app.SummaryTable1.Data{end, 2} = temp_SummaryTable3{end};
+                   end
+                   if (strcmp(app.StepOrPlayButton, 'step') && (i == nSubframes-1))
+                       app.Label.Text = {'Test stopped.'}; 
+                       app.PlayStopButton.Enable = 'on';                        
+                       app.StepButton.Enable = 'on';
+                       app.DocButton.Enable = 'on'; 
+                       app.GridButton.Enable = 'on';
+                       app.RefreshButton.Enable = 'on';
+                       app.TxDropDown.Enable = 'on';
+                       app.TxDropDown.Value = obj.PlutoTx.RadioID;
+                       app.RxDropDown.Enable = 'on';
+                       app.RxDropDown.Value = obj.PlutoRx.RadioID;
+                       app.dBPercentDropDown.Enable = 'on';
+                       app.TMNDropDown.Enable = 'on';
+                       app.BWDropDown.Enable = 'on';
+                       app.LOEditField.Enable = 'on';
+                       
+                       app.PBCHCheckBox.Enable = 'on';
```

Comment:
Setting all these properties looks like a duplicate from above. This should be a separate method